Cabal/Distribution/Simple/Build/PathsModule.hs

```diff
@@ -242,19 +224,10 @@ generate pkg_descr lbi clbi =
 
         paths_modulename = autogenPathsModuleName pkg_descr
 
-        get_prefix_stuff = get_prefix_win32 supports_cpp buildArch
+        get_prefix_stuff = get_prefix_win32
 
         path_sep = show [pathSeparator]
 
-        supports_cpp = supports_language_pragma
```

Comment:
tnose have to stay. We can build `Cabal` only with GHC-7.6+, but we should support building with GHC-7.0+. (i.e. via `cabal-install`).

Reply:
OK.

Reply:
Fixed.